ajax/libs/webicons/package.json

```diff
@@ -11,5 +11,6 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/adamfairhead/webicons.git"
-  }
+  },
+  "license": "CC-BY-3.0-US"
```

Comment:
@PeterDaveHello is this format ok?

Reply:
This is a valid CC license, just not in the SPDX list.